tests/test_diffusion.py

```diff
@@ -123,7 +123,7 @@ def execute_devito(ui, dx=0.01, dy=0.01, a=0.5, timesteps=500):
     dt = dx2 * dy2 / (2 * a * (dx2 + dy2))
     # Allocate the grid and set initial condition
     # Note: This should be made simpler through the use of defaults
-    u = TimeData(name='u', shape=(nx, ny), time_order=1)
+    u = TimeData(name='u', shape=(nx, ny), time_order=1, space_order=2)
```

Comment:
I don;t completly understand the implementation part, but I have to say I like seeing the time and space order as part of the field, as it shows what (mathematical) properties are attached to it . It makes the all API a lot more mathematically rigorous and is just one number to carry around.